data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard_generic.json

```
@@ -66,5 +67,22 @@
     ],
     "speaker_effect": { "effect": { "npc_add_var": "mission_Boris_mission_1_cleanup_asked", "value": "yes" } },
     "responses": [ { "text": "Well, thanks anyway.", "topic": "TALK_DONE" } ]
+  },
+  {
+    "id": "TALK_GUARD_FM_HELICRASH_ANECDOTE",
+    "type": "talk_topic",
+    "dynamic_line": [
+      "Sure yeah, I remember, I was on caravan duty when I saw it, it got caught in of those weird storms, as we started driving away I heard a big crash sound.",
```

```suggestion
      "Sure yeah, I remember, I was on caravan duty when I saw it, it got caught in of those weird storms, as we started driving away I heard a big crash sound.",
```
Hearing a sound is a bit redundant and would you even hear a crash if it's 20m underwater?